src/doc/unstable-book/src/compiler-flags/on-broken-pipe.md

```diff
@@ -0,0 +1,71 @@
+# `on-broken-pipe`
+
+--------------------
+
+The tracking issue for this feature is: [#97889]
+
+Note: The ui for this feature was previously an attribute named `#[unix_sigpipe = "..."]`.
+
+[#97889]: https://github.com/rust-lang/rust/issues/97889
+
+---
+
+The `-Zon-broken-pipe=...` compiler flag  can be used to specify how libstd shall setup `SIGPIPE` on Unix platforms before invoking `fn main()`. This flag is ignored on non-Unix targets. There are three variants:
```

Comment:
```suggestion
The `-Zon-broken-pipe=...` compiler flag can be used to specify how libstd shall setup `SIGPIPE` on Unix platforms before invoking `fn main()`. This flag is ignored on non-Unix targets. There are three variants:
```

Hmm, I wonder if a warning should be issued on the platforms that the flag doesn't have any effects on. Probably would be more annoying than helpful.

Should the default behavior be mentioned here immediately as well?

Reply:
I agree a warning of other platforms probably would be more annoying than helpful.

I also agree the docs can be improved, for example with regards to the default behavior and child processes, so I have made some restructurings and improvements now.